docs/content/operations/api-reference.md

```diff
@@ -230,6 +230,19 @@ Enables all segments of datasource which are not overshadowed by others.
 
 Enables a segment of a datasource.
 
+* `/druid/coordinator/v1/datasources/{dataSourceName}/markUnused`
```

Comment:
Please add the below two things:

- The segments would be disabled when this API is called, but then can be enabled by the coordinator if any loadRule matches.
- If an indexing task was running which read the same segments at the same time, the behavior is undefined. All segments would be disabled but the indexing task can read data from those segments and succeed. Please add a warning as well to not use any indexing task and this API at the same time.

Reply:
@jihoonson Should this be called out at the top somewhere as it applies to multiple queries? Or maybe a link shared between the queries to this callout?

Reply:
Hmm good question. I think it makes sense to call out at the top of all POST and DELETE sections of the coordinator API.

Reply:
> The segments would be disabled when this API is called, but then can be enabled by the coordinator if any loadRule matches.

This won't happen. Load rules don't enable segments that are marked as used = false. (The "load" part is meant to refer to "loading on a particular tier", not marking things as used.)

> Please add a warning as well to not use any indexing task and this API at the same time.

I would soften this warning to avoid using indexing tasks and this API at the same time for the same datasource and time chunk. (It's fine if the time chunks or datasource don't overlap)

Reply:
> This won't happen. Load rules don't enable segments that are marked as used = false. (The "load" part is meant to refer to "loading on a particular tier", not marking things as used.)

Oh yeah that's correct.

Reply:
adjusted the docs